realm/realm-library/src/main/java/io/realm/internal/Keep.java

```diff
@@ -28,5 +29,6 @@
  */
 @Retention(RetentionPolicy.CLASS)
 @Target(ElementType.TYPE)
+@Inherited
```

Comment:
No need to add this @Inherited.
It's my bad.

I tried to use inherited annotation for proguard, but proguard does not recognize it.

I'll remove this.